### PR TITLE
assignment 6.2

### PR DESCRIPTION
**Pull request by mubeen2710**
assignment 6.2, 7, 8

### CODE REVIEW

**Review by vdongre304 — APPROVED** [review]

**Review by Jestinxavier — APPROVED** [review]

File: mubeen/html/assign6/assign6.2.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    border-radius: 2px 8px;
+}
+h4 {
+    text-align-last: right;
+    border: solid rgb(95, 95, 194);
+    background-color: rgb(95, 95, 194);
+    border-radius: 2px 8px;
+}
+
+
```
Comment (Pawan1005) [inline mubeen/html/assign6/assign6.2.css]: remove extra lines

File: mubeen/html/assign7/home.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
+H1 {
```
Comment (Pawan1005) [inline mubeen/html/assign7/home.css]: make sure which tag is you using

File: mubeen/html/assign7/login.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <div class="login">
+        <form action="">
+            <label for="name">name:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</label>
+            <input type="text"> <br><br>
+            <label for="password">Password:</label>
+            <input type="password"> <br><br>
+            <center><input type="submit"></center>
+        </form>
+    </div>
+
```
Comment (Pawan1005) [inline mubeen/html/assign7/login.html]: remove extra line

File: mubeen/html/assign7/login.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            <label for="name">name:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</label>
+            <input type="text"> <br><br>
+            <label for="password">Password:</label>
+            <input type="password"> <br><br>
+            <center><input type="submit"></center>
+        </form>
+    </div>
+    
+</body>
+</html>
```
Comment (Pawan1005) [inline mubeen/html/assign7/login.html:25]: create proper indentation

File: mubeen/html/assign7/personal.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="home.css">
+    <title>Personal</title>
+</head>
+<body>
+    <center><h1>Job Portal</h1></center>
+    <a href="home.html">Home</a>
+    <a href="login.html">login</a>
+    <a href="profile.html">profile</a>
+
```
Comment (Pawan1005) [inline mubeen/html/assign7/personal.html]: remove extra line

File: mubeen/html/assign7/personal.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+            <label for="phoneno">Phone Number:</label>
+            <input type="number"><br><br>
+            <label for="address">Address:</label>
+            <input type="text"><br><br>
+            <input type="text"><br><br>
+            <input type="submit" style="position: absolute; left: 75%;">
+        </form>
+    </div>
+</body>
+</html>
```
Comment (Pawan1005) [inline mubeen/html/assign7/personal.html:34]: make proper indentation

File: mubeen/html/assign7/professional.html [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+                <label for="work">work Experience</label>
+                <input type="text"><br><br>
+                <label for="favorate">Favorate Field</label>
+                <input type="text"><br><br>
+                <input type="submit" style="position: absolute; left: 75%;">
+            
+        </form>
+    </div>
+</body>
+</html>
```
Comment (Pawan1005) [inline mubeen/html/assign7/professional.html:45]: make proper indentation
remove extra line

File: mubeen/html/assign7/profile.html [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+    <center><h1>Job Portal</h1></center>
+    <a href="home.html">Home</a>
+    <a href="login.html">login</a>
+    <a href="profile.html">profile</a>
+    <div class="profile">
+        <a href="personal.html">Add Personal details </a>
+        <a href="professional.html">Add professional details</a>
+    </div>
+</body>
+</html>
```
Comment (Pawan1005) [inline mubeen/html/assign7/profile.html:20]: make proper indentation

File: mubeen/html/assign8/assign9.css [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+
+.im1 {
+    float:left;
+    padding-right: 50px;
+}
+
+.im2 {
+    float: right;
+    padding-left: 50px;
+}
```
Comment (Pawan1005) [inline mubeen/html/assign8/assign9.css]: r3move extra lines

File: mubeen/html/assign8/assign9.html [hunk 10]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="assign9.css">
+        <title>Document</title>
+    </head>
+    <div>
```
Comment (Pawan1005) [inline mubeen/html/assign8/assign9.html]: body tag is missing

File: mubeen/html/assign8/assign9.html [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+            irure laborum. Non elit mollit non dolore mollit consequat in proident anim ex occaecat do labore anim. Mollit consectetur labore adipisicing
+            adipisicing cillum ea est sint et. Lorem culpa ipsum ipsum mollit ullamco quis amet ex. Cillum minim dolore do in velit anim. Culpa do eiusmod excepteur
+            veniam esse duis eiusmod eu labore. Fugiat et adipisicing ad ullamco. Dolore proident exercitation esse culpa enim consectetur labore mollit cupidatat.
+            Ea id sint eiusmod dolor minim velit laborum consequat excepteur non sunt ullamco culpa qui. Pariatur veniam Lorem aliqua incididunt elit minim labore
+           
+        </p></div>
+        <div>
+</div>
+    <body></body>
+</html>
```
Comment (Pawan1005) [inline mubeen/html/assign8/assign9.html]: make proper indentation
you have to use here float property here so do it and push again

Comment (Pawan1005) [inline mubeen/html/assign8/assign9.html]: and check file name also make it correct

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**